.github/CODEOWNERS

```diff
@@ -943,7 +943,7 @@
 # ServiceOwners:                                                   @davidemontanari @atastrophic @marcodalessandro @rohankhandelwal @riteshrao
 
 # PRLabel: %DevOps - Infrastructure
-/sdk/devops/Azure.ResourceManager.DevOpsInfrastructure/            @oolaoluwa
+/sdk/devopsinfrastructure/Azure.ResourceManager.*/                 @oolaoluwa
```

Comment:
Please add the `ServiceLabel` block for this label as well.

Reply:
Like this?